src/Destination/Adapter/RestApiOutput.php

```diff
@@ -118,15 +119,15 @@ private function call($config, $values = [])
             if ($this->output->isVerbose()) {
                 $this->output->writeln("<error>We'd got an error from your request</error>");
                 $this->output->writeln(
-                    sprintf("<error>%s</error>", $e->getMessage())
+                    sprintf('<error>%s</error>', $e->getMessage())
```

Comment:
Call to method `getMessage` from undeclared class `\GuzzleHttp\Exception\ClientException`